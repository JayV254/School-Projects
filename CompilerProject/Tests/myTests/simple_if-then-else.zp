int main() {
    int a;
    a = 10;

    if(a < 20) then {
        puts("The number is less than 10");
    }
    else {
        puts("The  number is greater than 20");
    }

    return 0;
}
