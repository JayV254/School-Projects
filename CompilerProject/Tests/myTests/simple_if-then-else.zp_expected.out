The number is less than 10

